src/onMessage.civet

```diff
@@ -1,35 +1,22 @@
-* as prettier from prettier
 { Message } from discord.js
-{ compile } from @danielx/civet
-{ createSnippet } from ./createSnippet.civet
+compile, { format } from ./compile.civet
```

Comment:
I don't think it's good practice to use default and non-default exports. Please make `convert` into a non-default export.